test/server/cards/01_SOR/events/SneakAttack.spec.ts

```diff
@@ -0,0 +1,30 @@
+describe('Sneak Attack', function() {
+    integration(function(contextRef) {
+        it('Sneak Attack ability\'s should play Sabine for 1 resource less and defeat it at the end.', function () {
+            contextRef.setupTest({
+                phase: 'action',
+                player1: {
+                    hand: ['sneak-attack', 'sabine-wren#you-can-count-on-me', 'obiwan-kenobi#following-fate'],
+                    groundArena: ['battlefield-marine'],
+                    spaceArena: ['cartel-spacer'],
+                    base: { card: 'administrators-tower', damage: 0 },
```

Comment:
Can remove the damage and just do `base: 'administrators-tower'`. Same for leader, undeployed is the default so it can just be `leader: 'luke-skywalker#faithful-friend'`